docs/_docs/Starters Academy (LDSSA)/06-Remote-Experience.md

```diff
@@ -0,0 +1,139 @@
+have much more control over the type of experience you have. In previous batches, we had in 
+person events that we required attendance to. This created an environment where social was 
+the default. Now it will be the opposite where isolated is the default. That's fine for 
+introverts and not so great for extroverts.
+
+Both experiences will use the same technology. The difference is how you use them. So let's 
+talk first about the tech and then how you can use these tools to craft the type of experience 
+that you want.
+
+## Your experience
+
+There are a few different types of experiences that you can have with this format. A social 
+one or a solitary one. The choice is yours but in the case that you are feeling lost or demotivated, 
+we strongly recommend the social experience. This is how you make the friends that will help you 
+through the course.
+
+### Experience 1: Solitary
+
+This one is easy. If you are overqualified for the course, are super confident that you can 
+breeze through all of the material, and are not a particularly social person then this is the 
+one for you. You leave the auditorium room open, pay attention when presentations are up, and 
+then just work away for the rest of the time.
+
+If you are an introverted person that is not particularly confident in your ability to complete 
+the course, the solitary experience will be very tempting. DON'T DO IT! This is a trap. It's a 
+nice, warm, safe, familiar blanket that will suffocate your chances of completing the course. 
+This is how you end up in your own head with a nice piece of imposter syndrome eating away at 
+your path to success. Bite the bullet and go for Experience 2.
+
+
+### Experience 2: Social
+
+Send direct messages to people. Jump around to all of the specific rooms. If a technical problem 
+presents itself, be proactive and collaborative in coming up with a solution. Invite someone that 
+interests you to a private video chat. If there's too many people in a specific room for it to 
+be useful, create another one and invite specific people to it. You'll spend less time on the 
+learning material but you'll spend more time problem solving with other people. Problem solving 
+with people is how real relationships are built. Having relationships with people in the course 
+are what will carry you through when you're doubting yourself. 
+
+There are going to be technical difficulties. We are a volunteer organization and this is the 
+first time we've done this. When there are problems, make it an opportunity to meet someone 
+somehow and treat it as an ice breaker. In doing this, you'll make a friend or two and that 
+friend or two will make the next 9 months so much better.
+
+
+
+## The tools
+
+### The google calendar 
+
+The source of truth for the schedule is [the batch 5 google calendar](https://calendar.google.com/calendar/u/2?cid=Y18yODE2azVnYXVscmw3MXVybnJwbGhjbW05c0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t). 
```

Comment:
asks login

Reply:
#239 added an issue to finalize calendar stuff